src/harm.cpp

```diff
@@ -80,19 +80,19 @@ bool Harm::GetRootPitch(TransPitch &pitch, unsigned int &pos) const
 {
     const Text *textObject = dynamic_cast<const Text *>(this->FindDescendantByType(TEXT, 1));
     if (!textObject) return false;
-    std::wstring text = textObject->GetText();
+    std::u32string text = textObject->GetText();
 
     if (text.length() > pos && text.at(pos) >= 'A' && text.at(pos) <= 'G') {
         int pname = (text.at(pos) - 'C' + 7) % 7;
         int accid = 0;
         for (pos++; pos < text.length(); pos++) {
-            if (text.at(pos) == L'𝄫')
+            if (text.at(pos) == U'\U0001D12B')
```

Comment:
Can we define those unicode literals together in one place and use constants (with proper names) instead of cryptic literals?

Reply:
This is what I did first but had to [change](https://github.com/rism-digital/verovio/commit/9bdecc24f4dc091e4b9380869cf23595f04d373e) it because MS Visual Studio does not like it:
https://ci.appveyor.com/project/LaurentPugin/verovio-r1t6l/builds/44834728

We will add them as comment for improving readability.

Reply:
https://github.com/rism-digital/verovio/pull/3027/commits/1174fe2567162e3e325e6af9fdb07c74087ef768